tests/Integration/Behaviour/Features/Scenario/Product/Combination/update_combination_stock.feature

```diff
@@ -0,0 +1,88 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-combination-stock
+@reset-database-before-feature
+@clear-cache-before-feature
+@update-combination
+@update-combination-stock
+Feature: Update product combination stock information in Back Office (BO)
+  As an employee
+  I need to be able to update product combination stock information from BO
+
+  Background:
+    Given language with iso code "en" is the default one
+    And attribute group "Size" named "Size" in en language exists
+    And attribute group "Color" named "Color" in en language exists
+    And attribute "S" named "S" in en language exists
+    And attribute "M" named "M" in en language exists
+    And attribute "L" named "L" in en language exists
+    And attribute "White" named "White" in en language exists
+    And attribute "Black" named "Black" in en language exists
+    And attribute "Blue" named "Blue" in en language exists
+    And attribute "Red" named "Red" in en language exists
+
+  Scenario: I update combination options:
+    Given I add product "product1" with following information:
+      | name[en-US] | universal T-shirt |
+      | is_virtual  | false             |
+    And product product1 type should be standard
+    And I generate combinations for product product1 using following attributes:
+      | Size  | [S,M]              |
+      | Color | [White,Black,Blue] |
+    Then product product1 should have following list of combinations:
+      | reference      | combination name        | attributes           | impact on price | final price | quantity | is default |
+      | product1SWhite | Size - S, Color - White | [Size:S,Color:White] | 0               | 0           | 0        | true       |
+      | product1SBlack | Size - S, Color - Black | [Size:S,Color:Black] | 0               | 0           | 0        | false      |
+      | product1Blue   | Size - S, Color - Blue  | [Size:S,Color:Blue]  | 0               | 0           | 0        | false      |
+      | product1MWhite | Size - M, Color - White | [Size:M,Color:White] | 0               | 0           | 0        | false      |
+      | product1MBlack | Size - M, Color - Black | [Size:M,Color:Black] | 0               | 0           | 0        | false      |
+      | product1MBlue  | Size - M, Color - Blue  | [Size:M,Color:Blue]  | 0               | 0           | 0        | false      |
+    And combination "product1SBlack" should have following stock details:
+      | combination stock detail | value      |
+      | quantity                 | 0          |
+      | minimal quantity         | 1          |
+      | low stock threshold      | 0          |
+      | low stock alert is on    | false      |
+      | location                 |            |
+      | available date           | 0000-00-00 |
```

Comment:
As commented before I'd prefer to leave this empty which would reflect more that we expect `null` Or if we want to be explicite about it use `null` as expected value in this scenario But using `0000-00-00` doesn't reflect the fact that the expected value returned in the query is really `null` and not a DateTime which value is 0